lib/project_types/script/layers/infrastructure/assemblyscript_task_runner.rb

```diff
@@ -34,7 +34,9 @@ def install_dependencies
         def dependencies_installed?
           # Assuming if node_modules folder exist at root of script folder, all deps are installed
           return false unless ctx.dir_exist?("node_modules")
-          check_if_ep_dependencies_up_to_date!
+          # NOTE: Temporarily disable this dependency check, as the latest EP version with guest serialization enabled
+          # has breaking changes on user script.
+          # check_if_ep_dependencies_up_to_date!
```

Comment:
Not sure if this would help actually. This requires CLI to be updated. However, does `use_msgpack: true` already get enabled in the latest CLI? @freynca

Reply:
`use_msgpack` originates from our toolchain; the CLI just passes it along from `metadata.json`